src/index.ts

```diff
@@ -84,7 +86,7 @@ module.exports = {
             preprocess: vuePreprocessor,
         },
         svelte: {
-            ...svelteParsers.svelte,
+            ...svelteParsers.parsers.svelte,
```

Comment:
The only branch of createSvelteParsers where parsers is defined is the successful try case.

This will cause a type error for anyone without prettier-plugin-svelte.

Reply:
The fix is coming soon and I should never release anything on a Friday!